etl/steps/data/garden/wb/2023-07-10/education.py

```diff
@@ -73,21 +75,19 @@ def run(dest_dir: str) -> None:
     tb["normalized_hci"] = tb["HD.HCI.HLOS"] / max_value
 
     # Combine recent literacy estimates and expenditure data with historical estimates from a migrated dataset
-    tb, combined_literacy_description, combined_expenditure_description = combine_historical_literacy_expenditure(tb)
+    tb = combine_historical_literacy_expenditure(tb, tb_literacy, tb_expenditure)
 
     # Compare two columnst that seem to have identical indicies (if values are the same then remove)
     if tb["SE.XPD.TOTL.GD.ZS"].equals(tb["SE.XPD.TOTL.GD.ZS."]):
         # If they are the same, drop one of the columns
         tb.drop("SE.XPD.TOTL.GD.ZS.", axis=1, inplace=True)
     else:
         print("The columns are not the same.")
```

Comment:
Instead of printing, you should assert things. Assert that the columns are identical, then drop one of them. If it changes in the next update, you will not have any way to miss it.